test/indexer/test_hca_indexer.py

```diff
@@ -60,11 +60,15 @@ def setUpModule():
 
 class TestHCAIndexer(IndexerTestCase):
 
-    def _get_all_hits(self):
+    def _get_all_hits(self, verify_lists_sorted=True):
         hits = scan(client=self.es_client,
                     index=','.join(self.get_hca_indexer().index_names()),
                     doc_type="doc")
-        return list(hits)
+        hits = list(hits)
+        if verify_lists_sorted:
+            num_lists_checked = self._verify_sorted_lists(hits, [])
```

Comment:
The verifier needs to be called separately for every hit. That makes the assertion a little stronger.